dll.py

```diff
@@ -0,0 +1,40 @@
+class Node(object):
+
+    def __init__(self, vertex, prev, next):
+        self.vertex = vertex
+        self.prev = prev
+        self.next = next
+
+
+class DoubleLinkedList(object):
+    firstNode = None
```

Comment:
Misschien netter om dit in een `__init__` te zetten?

Reply:
Suggestie: 
```
def __init__(self, node: "Vertex"):
  self._first_node = node
  self._last_node = node
```
Nb. conventie is om attribute namen te starten met een `_` en snake case (blijkbaar)